test/integration/framework/controlplane_utils.go

```diff
@@ -80,8 +80,18 @@ func (alwaysAllow) Authorize(ctx context.Context, requestAttributes authorizer.A
 	return authorizer.DecisionAllow, "always allow", nil
 }
 
-// alwaysEmpty simulates "no authentication" for old tests
-func alwaysEmpty(req *http.Request) (*authauthenticator.Response, bool, error) {
+// unsecuredUser simulates requests to the unsecured endpoint for old tests
+func unsecuredUser(req *http.Request) (*authauthenticator.Response, bool, error) {
```

Comment:
only return the privileged unsecured user if the request didn't attempt to provide any authentication info. if req has an `Authorization` header, `return nil, false, nil`

Reply:
fixed